frontends/ol-components/src/components/Card/Card.tsx

```diff
@@ -180,27 +192,22 @@ export const useClickChildHref = (
         anchor.click()
       }
     },
-    [href, onClick],
+    [onClick],
   )
 }
 
 type CardProps = {
   children: ReactNode[] | ReactNode
   className?: string
   size?: Size
-  /**
-   * If provided, the card will render its title as a link.
-   *
-   * Clicks on the entire card can be forwarded to the link via `forwardClicksToLink`.
-   */
-  href?: string
   /**
    * Defaults to `false`. If `true`, clicking the whole card will click the
-   * href link as well.
+   * child anchor with data-card-link="true".
```

Comment:
Previously, if `forwardClicksToLink` was true, clicking the whole card would click the child anchor with the card's `href`.

This turned out to be a bit fragile with NextJS. For example, on the `/units` listing page, the backend currently returns `channel_url` values like `http://localhost:8062/c/unit/mitx/`. But NextJS standardizes the href to NOT have a trailing slashing: `http://localhost:8062/c/unit/mitx`. 

We could try to trim the trailing slash, but that seems complicated for things like `/some/path/?foo=bar` vs `/some/path?foo=bar`. So instead, let's just find the relevant anchor by explicitly marking it with `data-card-link`.

Consequently, the `Card` no longer needs the href, just the `Card.Title`.

Reply:
Something I'm noticing is that previously, the unit cards were wrapped in an `a` tag when this was rendered in the browser. Now, following the link seems to be handled by Javascript. It doesn't make logical sense to me either that an `href` property would be set on the title, as the whole card is supposed to be clickable?